Automattic/wp-calypso
# Plans: Add beginning of `PlanOverview`, `PlanFeatures`, and `PlanStatus`

This pull request addressed part of https://github.com/Automattic/wp-calypso/issues/1151 by introducing a new variant of the `Plans` page. This new `Plan Overview` page is displayed when the user has subscribed to a free trial.

<img width="744" alt="screen shot 2015-12-09 at 4 10 45 pm" src="https://cloud.githubusercontent.com/assets/1130674/11698940/86842af6-9e8f-11e5-961d-3a2191bd4b98.png">

Three new components are added in this PR: `PlanOverview`, `PlanFeatures`, and `PlanStatus`.
- `PlanOverview` is used to render `/plans/:site` when `:site` has a free trial.
- `PlanStatus` implements a `<ProgressBar />` to show the amount of time remaining in a trial. To be implemented in a separate PR:
   - The 'Purchase Now' button.
   - The states where a plan is near to expiring or expired (with a red progress bar).
- `PlanFeatures` is a stub component and will be implemented in a separate PR.

**Testing**
- Visit `/plans/:site` and start a premium or business free trial.
- Visit `/plans/:site` and assert that you see `PlanOverview` as above.
